app/build.gradle

```diff
@@ -2,6 +2,8 @@ plugins {
     id "com.diffplug.gradle.spotless" version "3.26.1"
```

Comment:
How do you want to proceed with this PR, as there are still some todo comments inside the code?

Reply:
You are right about the TODO comments, they were there as a reminder for possible improvement. 
To resume :
* A function to force to unregister an app: I don't think I have even needed it during my tests. There is still a workaround if it is really needed: wiping app data.
* As discussed, a notification to let the user to chose :
	* If we accept an app registration
	* If we delete the app on unregistration
* Automatic notification deletion (like issue #138)
* Implementing a special notification on URL change. As it happens very rarely, re-register an app is a good way to deal with it for the moment.

Should I delete the comments and we will write an issue as a feature request for these ? Implementing the notification for user acknowledge, and write a feature request for the others ? Something else ?

I'm committing the deletion of these lines for the moment